srcpkgs/alass/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'alass'
+pkgname=alass
+version=2.0.0
+revision=1
+build_style=cargo
```

Comment:
this will make it install properly
```suggestion
build_style=cargo
make_install_target="--path alass-cli"
```

Reply:
Thanks again for the help!! With the new `make_install_target`, I'm still getting the same error "`found a virtual manifest at `/builddir/alass-2.0.0/Cargo.toml` instead of a package manifest`" as before. It seems like the path should be right, but this is my first time using cargo.

Reply:
hm weird, it worked for me when I tried it locally